src/olympia/addons/management/commands/process_addons.py

```diff
@@ -153,6 +159,8 @@ def handle(self, *args, **options):
         pks = (addon_manager.filter(*task['qs'])
                             .values_list('pk', flat=True)
                             .order_by('id'))
+        if options.get('limit'):
+            pks = pks[:options.get('limit')]
```

Comment:
`options['limit']` should be cast to an integer, if you call this manually you should get some kind of error IMHO.

By default arguments are only parsed as strings.

To verify:

```python
> /code/src/olympia/addons/management/commands/process_addons.py(172)handle()
    171         import ipdb; ipdb.set_trace()
--> 172         if 'pre' in task:
    173             # This is run in process to ensure its run before the tasks.

ipdb> options
{'task': 'extract_webextensions_to_git_storage', 'settings': None, 'pythonpath': None, 'verbosity': 1, 'traceback': False, 'ids': None, 'with_deleted': False, 'limit': '5', 'no_color': False}
ipdb> options['limit']
'5'
```

Reply:
added the `type` param in https://github.com/mozilla/addons-server/pull/9699/commits/385d572c9fe0a4921c617162bfc377573d205d0a